jupyter-book/concluding_remarks.md

```diff
@@ -0,0 +1,261 @@
+
+### Technical craft is not all
+
+We gave methodological elements, but these are not enough to always have
+solid conclusion from a statistical standpoint.
+
+
+Once you know how to run the software, the biggest challenges are understanding the data, its shortcomings, and what can and cannot be concluded from an analysis
+  - Automating machine learning does not solve data science
+  - Domain knowledge and critical thinking about the data
+
+
+### How the predictions are used
+
+When designing a machine-learning system, we need to think about
+how the predictions are used.
+
+Errors mean different things in different application contexts.
+
+- Operational risk:
+    - Advertisement: errors are often harmless
+    - Medicine: errors can kill
+- Operational logic: Better a false detection or a miss?
+    - Detecting brain tumors:
+        - If a patient is sent to surgery: false detections are very dangerous
+        - If a patient given an MR scan to confirm the detection:
+	  misses should be avoided, as an MR scan is harmless
+
+The predictions may modify how the system is functions:
+  - Predicting who will benefit from an hospital stay may overcrowd some units of the hospital, and thus change the positive impact of hospitals on inpatients
+
+### Choice of the output/the labeled dataset
+
+- What we chose to predict is a very loaded choice
+- Interesting labels are often hard to get, focusing on the "easy" ways
+  of accumulating labels comes with biases
+- Our target may be a proxy of the quantity of interest
+
+
+### Biases in the data
+
+All data come with biases.
+
+- The data may not reflect the ground truth
+    - Disease monitoring is function of testing policy
+    - It may change with time, it may be uneven across the population (eg higher quality data for rich people)
+- The state of affaires may not be the desired one
+    - For equal qualifications and responsibilities, women are typically payed less than men. A learner will pick this up and amplify inequalities
+
+### Prediction models versus causal models
+
+Machine learner models are not driven by causal mechanisms.
+
+- For example people that go to the hospital die more than people who do not:
+    - Naive data analysis might conclude that hospital are bad for health
+    - The fallacy is that we are comparing different populations: people
+      who go to the hospital typically have a worse baseline health than
+      people who do not.
+- Another example: having a heart pressure greater than a threshold
+  may trigger specific care which is good. A learner will pick up
+  above-threshold heart pressure as predictor of a health improvement
+- In pure predictive settings, these informations are beneficial for the
+  predictions. However:
+ - they should not be trusted when designing interventions
+ - predictive models built on such non-causal information may be brittle
+   to changes of operational procedures
+ - interpretation is subject to caution
```

Comment:
```suggestion
 - interpretation must be subject to caution
```

Reply:
I think that "is subject to" is preferable

Reply:
Alright, not a strong opinion on this one, I thought we could use "must to" with the intention to "strongly recommend"

Reply:
Maybe we could have some kind of conclusion to this section:

Quantifying the expected causal effect of a real world intervention is often best
achieved by randomized control trials or A/B tests. In comparison, purely
data-driven observational studies are often tricky to get right in practice.
If you want to learn more about this topic, you might want to explore the
following resources:

- https://github.com/microsoft/EconML
- https://github.com/uber/causalml